website/docs/r/entity_tags.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_entity_tags"
+sidebar_current: "docs-newrelic-resource-entity-tags"
+description: |-
+  Create and manage tags for a New Relic One entity.
+---
+
+# Resource: newrelic\_entity\_tags
+
+Use this resource to create, update, and delete tags for a New Relic One entity.
+
+A New Relic Personal API key is required to provision this resource.  Set the `api_key`
+attribute in the `provider` block or the `NEW_RELIC_API_KEY` environment
+variable with your Personal API key.
```

Comment:
Not a blocker, but thoughts on an additional "for more information" sentence linking to the "Configuring the Provider" page?

Reply:
👍